### DISCUSSION TITLE
Internal server error 500 on asset with UTF-8 file names

### DISCUSSION THREAD

**Discussion by gintsn** [requarks/wiki#5218] [Error / Bug Report] [answered]
Hi, 
It is possible to upload asset file with UTF-8 names (like `šķūrē.pdf`) and publish them in pages, but when we try to download file from page, we got error 500:  `Internal Server Error`

```
OS:  Ubuntu 18.04.6 LTS
wiki.js version:  wikijs-2.5.274
db: Sqlite3
```
With latest Wiki.JS version `wikijs-2.5.278` the pages are working as expected without error.

**Comment by NGPixel** [comment, marked as answer]
Closing as this is already fixed in 2.5.277...
